cypress/e2e/inventory/moving-holdings.cy.js

```diff
@@ -0,0 +1,98 @@
+import getRandomPostfix from '../../support/utils/stringTools';
+import permissions from '../../support/dictionary/permissions';
+import testTypes from '../../support/dictionary/testTypes';
+import devTeams from '../../support/dictionary/devTeams';
+import InventorySearchAndFilter from '../../support/fragments/inventory/inventorySearchAndFilter';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import Users from '../../support/fragments/users/users';
+import TopMenu from '../../support/fragments/topMenu';
+import ItemView from '../../support/fragments/inventory/inventoryItem/itemView';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+import InteractorsTools from '../../support/utils/interactorsTools';
+import InventoryInstancesMovement from '../../support/fragments/inventory/holdingsMove/inventoryInstancesMovement';
+import InventoryHoldings from '../../support/fragments/inventory/holdings/inventoryHoldings';
+
+// TO DO: remove ignoring errors. Now when you click on one of the buttons, some promise in the application returns false
+Cypress.on('uncaught:exception', () => false);
+
+let userId;
+const item = {
+  instanceName: `Inventory-first-${Number(new Date())}`,
+  barcode: `123${getRandomPostfix()}`,
+  firstHoldingName: '',
+  holdings: [],
+};
+
+const secondItem = {
+  instanceName: `Inventory-second-${getRandomPostfix()}`,
+  barcode: `123${getRandomPostfix()}`,
```

Comment:
please update 123 to 124 for example because there can be a situation when two items can have the same barcode

Reply:
done